src/block/tile/BrewingStand.php

```diff
@@ -45,11 +53,11 @@ class BrewingStand extends Spawnable implements Container, Nameable{
 	private $inventory;
 
 	/** @var int */
-	private $brewTime = 0;
+	private $brewTime;
 	/** @var int */
-	private $maxFuelTime = 0;
+	private $maxFuelTime;
 	/** @var int */
-	private $remainingFuelTime = 0;
+	private $remainingFuelTime;
```

Comment:
These defaults need to stay. They might not be initialized if the tile wasn't loaded from disk.